app/models/user.rb

```diff
@@ -1,8 +1,13 @@
 class User < ApplicationRecord
 
+  EMAIL_REGEXP = /^[\w\.-]+@([\w-]+\.)+[\w-]{2,4}$/.freeze
```

Comment:
add a new line

Reply:
this is not a valid email regex. check on google

Reply:
done